docs/data/data-grid/list-view/list-view.md

```diff
@@ -0,0 +1,25 @@
+---
+title: Data Grid - List view
+---
+
+# Data Grid - List view [<span class="plan-pro"></span>](/x/introduction/licensing/#pro-plan 'Pro plan')
+
+<p class="description">Display data in a single-column list view. Can be used to present a more compact grid on smaller screens and mobile devices.</p>
+
+List view can be enabled by providing the `unstable_listView` prop.
+
+Unlike the default grid view, the list view makes no assumptions on how data is presented to end users.
+
+In order to display data in a list view, a `unstable_listColumn` prop must be provided with a `renderCell` function.
+
+{{"demo": "ListView.js", "bg": "inline"}}
+
+## Advanced usage
+
+The list view feature can be combined with [custom subcomponents](/x/react-data-grid/components/) to provide an improved user experience on small screens.
+
+{{"demo": "ListViewAdvanced.js", "bg": "inline", "hideToolbar": true}}
```

Comment:
Providing access to the code here too could be nice, unless there's a catch that I'm not aware of?
```suggestion
{{"demo": "ListViewAdvanced.js", "bg": "inline"}}
```

Reply:
The demo code viewer is not showing the full list of files currently. I'm going to create a follow up issue. See the [Notion doc](https://www.notion.so/x-grid-Mobile-view-for-Data-Grid-b6228123c8a64352ba2c355c46e4e4ff?d=112cbfe7b66080f0ba5f001cfc35bf8d&pvs=4#5c1fee4d0ec04e629ddf32270f70a1e8). For now we are just going to link to a CodeSandbox below.